app/src/main/java/com/ikarabulut/Board.java

```diff
@@ -51,9 +51,10 @@ public List<ArrayList> getBoard() {
         return board;
     }
 
+    //TODO: Remove this, its not used
```

Comment:
This was for the `getRows() && getColumns()` getter methods in `Board` I felt that they were redundant since `GameSettings` technically sets the number for rows and columns. I had to make a change in `Board` `generateNextBoard()` to use `GameSettings` getter instead of `Boards` which I did now and will push in my next commit